flang/lib/Lower/OpenMP.cpp

```diff
@@ -55,8 +55,12 @@ static void createBodyOfOp(Op &op, Fortran::lower::FirOpBuilder &firOpBuilder,
   firOpBuilder.createBlock(&op.getRegion());
   auto &block = op.getRegion().back();
   firOpBuilder.setInsertionPointToStart(&block);
-  // Ensure the block is well-formed.
-  firOpBuilder.create<mlir::omp::TerminatorOp>(loc);
+  if (isa<omp::WsLoopOp>(op.getParentOp())) {
+    mlir::ValueRange results;
+    firOpBuilder.create<mlir::omp::YieldOp>(loc, results);
+  } else
```

Comment:
You should add braces for the `else` block to keep it uniform with the `if` block. 

https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements